iml-agent/src/action_plugins/ldev.rs

```diff
@@ -60,148 +65,233 @@ pub async fn create(entries: Vec<LdevEntry>) -> Result<(), ImlAgentError> {
                 label: "zfsmo-OST0007".into(),
                 device: "zfs:ost7/ost7".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss1".into(),
                 failover: Some("oss2".into()),
                 label: "zfsmo-OST0008".into(),
                 device: "zfs:ost8/ost8".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss1".into(),
                 failover: Some("oss2".into()),
                 label: "zfsmo-OST0009".into(),
                 device: "zfs:ost9/ost9".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST000a".into(),
                 device: "zfs:ost10/ost10".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST000b".into(),
                 device: "zfs:ost11/ost11".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST000c".into(),
                 device: "zfs:ost12/ost12".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST000d".into(),
                 device: "zfs:ost13/ost13".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST000e".into(),
                 device: "zfs:ost14/ost14".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST000f".into(),
                 device: "zfs:ost15/ost15".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST0010".into(),
                 device: "zfs:ost16/ost16".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST00011".into(),
                 device: "zfs:ost17/ost17".into(),
+                fs_type: FsType::Zfs,
             },
             LdevEntry {
                 primary: "oss2".into(),
                 failover: Some("oss1".into()),
                 label: "zfsmo-OST00012".into(),
```

Comment:
While here, could you remove extra 0s? :) 
```suggestion
                label: "zfsmo-OST0012".into(),
```

Reply:
Sure, I removed it on all of them.